server/hotrod/src/main/java/org/infinispan/server/hotrod/Encoder2x.java

```diff
@@ -331,7 +332,7 @@ public void writeResponse(Response response, ByteBuf buf, EmbeddedCacheManager c
             BulkGetResponse r = (BulkGetResponse) response;
             if (isTrace) log.trace("About to respond to bulk get request");
             if (r.status == OperationStatus.Success) {
-               try (CloseableIterator<Map.Entry<byte[], byte[]>> iterator = r.entries.iterator()) {
+               try (CloseableIterator<Map.Entry<byte[], byte[]>> iterator = (CloseableIterator<Map.Entry<byte[], byte[]>>) r.entries.iterator()) {
```

Comment:
is this cast needed?

Reply:
hmmm it was probably not compiling for some reason in my laptop. Need to recheck